src/btree/bt_vrfy.c

```diff
@@ -217,7 +217,7 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
 			WT_WITH_PAGE_INDEX(session,
 			    ret = __verify_tree(session, &btree->root, vs));
 
-			WT_TRET(__wt_cache_op(session, NULL, WT_SYNC_DISCARD));
+			WT_TRET(__wt_cache_op(session, NULL, WT_SYNC_CLOSE));
```

Comment:
@michaelcahill: thinking about this last night, it seems to me we should remove this line of code: there's no reason to believe the pages read into cache for one checkpoint won't be useful in the subsequent verify of a different checkpoint, so I think this is unnecessary complexity to no significant advantage, we should just let eviction do its job.